hszinc/zincparser.py

```diff
@@ -419,7 +420,7 @@ def _parse_datetime(toks):
             hs_date, hs_time, hs_coord, hs_number, hs_null, hs_marker,
             hs_remove, hs_bool]).setName('scalar')
 hs_scalar_3_0 <<= Or([hs_ref, hs_bin, hs_str, hs_uri, hs_dateTime,
-            hs_date, hs_time, hs_coord, hs_number, hs_null, hs_marker,
+            hs_date, hs_time, hs_coord, hs_number, hs_null,hs_na, hs_marker,
```

Comment:
Minor nit here, `hs_na` should really have a space before it to be consistent with the others… I'll fix this after the merge. :-)